.vib/harbor-exporter/goss/harbor-exporter.yaml

```diff
@@ -0,0 +1,15 @@
+group:
+  harbor:
+    exists: true
+user:
+  harbor:
+    exists: true
+command:
+  # Ensure a set of directories exist and the non-root user has write privileges to them
+  check-directories-exist-with-user:
+    exec: ls -dl /etc/ssl/certs /etc/pki/tls/certs/ /var/log/jobs 2>/dev/null | grep "drwxrwxr-x.*harbor"
```

Comment:
This check will succeed if just one dir exists and I think `/var/log/jobs` is always present, so the test will pass even if both `/etc/ssl/certs /etc/pki/tls/certs/` are not present, which is not what we want.

In this case, I suggest adding a separate [file](https://github.com/goss-org/goss/blob/master/docs/manual.md#file) check for `/var/log/jobs`.